torch/_dynamo/utils.py

```diff
@@ -251,16 +261,20 @@ def dynamo_timed(
     fail_type: Optional[str] = None
     fail_reason: Optional[str] = None
     time_spent = float("-inf")
+    if phase_name == "entire_frame_compile":
+        reset_chromium_events()
```

Comment:
What's going on here? (Is it required to hardcode this specific phase name?)

Reply:
Honestly we don't even need this, I had it before when I wanted to reset the uuid every compile. Gonna remove